### PR TITLE
CNF-9624: Add IBU CR to ztp/source-crs

### PR DESCRIPTION
**Pull request by sakhoury**
Add IBU CR to ztp/source-crs and set the stage to Idle.

/cc @imiller0 @Missxiaoguo @donpenney

### CODE REVIEW

**Review by imiller0 — APPROVED** [review]
/lgtm

File: ztp/source-crs/ImageBasedUpgrade.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+apiVersion: lca.openshift.io/v1alpha1
+kind: ImageBasedUpgrade
+metadata:
+  name: upgrade
+  annotations:
+    ran.openshift.io/ztp-deploy-wave: "10"
```
Comment (jc-rh) [inline ztp/source-crs/ImageBasedUpgrade.yaml:6]: Would this CR ever be in a policy that's part of ZTP?

Comment (jc-rh) [reply ztp/source-crs/ImageBasedUpgrade.yaml:6]: Remove the annotation to avoid confusion then

Comment (sakhoury) [reply ztp/source-crs/ImageBasedUpgrade.yaml]: Done

Comment (sakhoury) [reply ztp/source-crs/ImageBasedUpgrade.yaml]: @jc-rh the [ci job](https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift-kni_cnf-features-deploy/1727/pull-ci-openshift-kni-cnf-features-deploy-master-ci/1743371685165273088) is failing with the following validation error:
```
Error: missing annotation 'ran.openshift.io/ztp-deploy-wave' in ./source-crs/ImageBasedUpgrade.yaml
```

**Pull request merged**